python/src/opendp/context.py

```diff
@@ -920,8 +1014,11 @@ def _caster(measurement):
             raise ValueError(f"alpha ({alpha}) must be in [0, 1)")  # pragma: no cover
         delta_zCDP, delta_inf = delta * (1 - alpha), delta * alpha
 
-        rho = _translate_measure_distance((epsilon, delta_zCDP), from_measure, zero_concentrated_divergence())
+        rho = _translate_measure_distance(
+            (epsilon, delta_zCDP), from_measure, zero_concentrated_divergence()
+        )
         return rho, delta_inf
-        
 
-    raise ValueError(f"Unable to translate distance from {from_to[0]} to {from_to[1]}")  # pragma: no cover
+    raise ValueError(
+        f"Unable to translate distance from {from_to[0]} to {from_to[1]}"
+    )  # pragma: no cover
```

Comment:
Slight preference for keeping reformating in separate PR.

Reply:
But don't worry about it here!

Reply:
It's a mistake when it happens. Sorry for the noise.